### ISSUE TITLE
Lots of buffering and videos not playing

### ISSUE THREAD

**Issue by mkil5** [yuliskov/SmartTube#1511]
I am using an Nvidia Shield with both SmartTube and STN Beta both installed and am having the same problems on both.

This started about a month ago, before this everything was just fine.  I typically play my videos with 1.5-2x speed, depending on the content producer.  Now my videos are constantly buffering if they play.  Many times the video will just hang on the loading screen.  Sometimes I can restart the video and it will work, most of the times it will not.  I have changed my video settings since I saw that some videos were running in 2160p, I went to the 1080p vp9 since I do not need 4k and thought it could be a problem with the bandwidth being too much (it shouldn't be 200mbps on ethernet and I never had a problem until about a month ago).

I have reinstalled the app, reset data.  Done updates.  These problems have happened in the past but an update usually takes care of them really fast.  So I was wondering if anyone else is having this problem or what is going on?

Thanks

**Comment by edgan** [comment]
8nuave seen this issue two. Sometimes restarting the app works. Sometimes rebooting the Shield works. Sometimes neither works. It seems to be somewhat dependent on the video.

I don't think it is the same issue, but a lot of the randomness of the bug remembers me of my skipping video bug.

https://github.com/yuliskov/SmartTubeNext/issues/1509

**Comment by zizebra** [comment]
Same here I was pulling my hair thinking my internet link is the problem. It's been terrible since the last update a couple of days ago. Iam running on shield tv

**Comment by ytsejam1138** [comment]
Same problems. Seemed to start with the latest update, but I reinstalled the previous stable build and I am still having the pausing/stuttering issues on most videos.

**Comment by norbank** [comment]
Similar issue on NVidia shield, only for 4k content, play normal at first, has up to 20s buffer time, after 2 to 5 mins it stops buffering, buffer time down to 0s, then it stop play, buffering only 1 or 2 seconds then play, and repeat. 
Not my bandwidth issue, because native YouTube app works fine.

**Comment by waseem852** [comment]
Have the same issue.

**Comment by jdgremsjr** [comment]
I just started to have this issue on my 3 fire devices, I have one Toshiba with the fire OS built in, a fire cube and a fire stick they all appear to be doing it. I tested my speed and I'm connecting at 500mbps, I've rebooted the devices, cleared the cache, uninstalled the program and re-installed. this started today right before I installed the update for 15.76. I'm just at a loss. None of my other apps are doing this. So I don't think it is a network problem. Please help.

**Comment by crypet** [comment]
Got the same issue today on both of my nvidia shield. Found that when I logout of my account, it works fine. As soon as I logon, the issue is back. 

I guess YouTube has added some throttling on my account when using smarttube. 

Native YouTube app works fine. 

I hope this will be fixed in the next update.

**Comment by shopkeeper1970** [comment]
> Got the same issue today on both of my nvidia shield. Found that when I logout of my account, it works fine. As soon as I logon, the issue is back.
> 
> I guess YouTube has added some throttling on my account when using smarttube.
> 
> Native YouTube app works fine.
> 
> I hope this will be fixed in the next update.

Having the same issue today too. Had to reduce the video quality to 360 to get rid of the buffering. Picture still watchable.

**Comment by Dom51487** [comment]
Happened to me too this afternoon. Was working fine before and then out of the blue video started buffering. Tried force closing the app but still buffering. Tried uninstalling/reinstalling but still buffering. I have to use the native YouTube app now with the ads. :(

**Comment by MichaelOultram** [comment]
> Got the same issue today on both of my nvidia shield. Found that when I logout of my account, it works fine. As soon as I logon, the issue is back. 
> 
> I guess YouTube has added some throttling on my account when using smarttube. 
> 
> Native YouTube app works fine. 
> 
> I hope this will be fixed in the next update. 

I have the same experience and it also just started today. I use a Nvidia shield and can replicate the logged in vs logged out behaviour. SmartTube 15.62

**Comment by chas901** [comment]
This has just started happening to me today! Worked fine last night for me and both my kids fire sticks. First noticed with my son's fire stick, I thought it may have been the hot weather we're having in the UK atm so I restarted the router and WiFi AP.. still getting 500mbps down and 200-300mbps over WiFi so wasn't that. I checked my daughters and mine and both are doing the same thing, buffering playing 2 seconds then buffering again and so on. 

So..
- All were working last night.
- We were all on different versions of the app, so it's not that.
- Official YouTube app works fine.
- YouTube Vanced still works fine on my phone, no issues with skipping ads and sponsors.
- Tried the buffering fix in the developer options but makes no difference.

YouTube must be doing this some how! 🤔

I really hope someone is able to figure this out because I tried the official YouTube app and so far every video has started with two 20 second ads, 3 years of no ads on YouTube I'm so used to it now that I actually won't watch YouTube because of the ads! 🤦😅

**Comment by delwark17** [comment]
Same issue started today (8/11) for me on my Android tv. Even before I updated to version 15.78. No buffering on any other apps, including native YouTube or YouTube vanced. Fairly certain there's no connection issues either. Agree that this might be a YouTube throttling thing. Issue seems to get a little better when I sign out and use the app? But not by much. 

Hoping this gets fixed soon. Having no ads has spoiled me, I doubt I can go back (although that would probably encourage me to be more productive)

**Comment by jdgremsjr** [comment]
Just so everyone knows, there was an update. 15.79 and that has corrected the problem. Thank you to the developer. You were on it quickly and solved the problem.

**Comment by crypet** [comment]
Yes, update to version 15.78 fixed the issue for me. Thank you devs! 🙏

**Comment by Erdoboy48** [comment]
I have updated to 15.79 and it's still buffering for me. YouTube works fine. I am using a fire tv gen 1 box

**Comment by Erdoboy48** [comment]
Working ok for me now. All very strange indeed. Cheers.

**Comment by jordanade** [comment]
I just experienced this with 15.81

**Comment by Chealer** [comment]
@mkil5 : Which exact versions does this affect? Did you read the answers of the 2 relevant FAQ questions? Does it happen at normal speed? Which videos are affected?

**Comment by waseem852** [comment]
Same thing happens with me either I am on VPN or I am running on 1.5x. so I think that should be normal. Not facing the issue when I am on 1x or No VPN.
